torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -202,6 +202,7 @@ def __init__(
         configs,
         save_cache_hook,
         mutated_arg_names: List[str],  # see [Note: clone mutated buffers]
+        reset_to_zero_arg_names: List[str],
```

Comment:
Can we add this at the end with the default `None` for BC? And handle it similar to `inductor_meta`.